lib/routes/cdn.js

```diff
@@ -68,6 +68,7 @@ module.exports =
         }
       },
       preHandler: fastify.auth([fastify.verifyAuthentication]),
```

Comment:
`verifyAuthentication` this mechanism is also going to need to have security logging added to it in some way.

Reply:
Yes indeed. This is what we handle in our private repo indeed.